extension-config.json

```diff
@@ -91,6 +91,36 @@
       "id": "pl-PL",
       "path": "frontend/locale/pl-PL.json",
       "type": "translations"
+    },
```

Comment:
changes for `@shopgate/user-privacy` need to be done inside the `pwa` repository https://github.com/shopgate/pwa/blob/master/extensions/%40shopgate-user-privacy/extension-config.json

Reply:
ok, closed this PR